docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
We will need to fully document the use case of running the container using docker-compose.

Reply:
Probably yes. But due to lack of time, it might not happen.